examples/c/ex_col_store.c

```diff
@@ -0,0 +1,555 @@
+#include <test_util.h>
+
+#define N_DATA 100
+
+static const char *home;
+
+typedef struct {
+    char day[5];
+    uint16_t hour;
+    uint8_t temp;
+    uint8_t humidity;
+    uint16_t pressure;
+    uint8_t wind;
+    uint8_t feels_like_temp;
+    uint16_t loc_lat;
+    uint16_t loc_long;
+    char country[5];
+} WEATHER;
+
+uint8_t celsius_to_fahrenheit(uint8_t temp_in_celcius);
+void update_celsius_to_fahrenheit(WT_SESSION *session);
+void print_temp_column(WT_SESSION *session);
+void print_all_columns(WT_SESSION *session);
+void chance_of_rain(WT_SESSION *session);
+void generate_data(WEATHER *w_array);
+void remove_country(WT_SESSION *session);
+void search_temperature(WT_SESSION *session);
+void average_data(WT_SESSION *session);
+bool find_min_temp(WT_SESSION *session, uint16_t start_time, uint16_t end_time, int *result);
+bool find_max_temp(WT_SESSION *session, uint16_t start_time, uint16_t end_time, int *result);
+
+void
+print_all_columns(WT_SESSION *session)
+{
+    WT_CURSOR *cursor;
+    const char *country;
+    int ret;
+    uint64_t recno;
+    const char *day;
+    uint16_t hour, pressure, loc_lat, loc_long;
+    uint8_t humidity, temp, wind, feels_like_temp;
+
+    error_check(session->open_cursor(session, "table:weathertable", NULL, NULL, &cursor));
+    while ((ret = cursor->next(cursor)) == 0) {
+        error_check(cursor->get_key(cursor, &recno));
+        error_check(cursor->get_value(cursor, &day, &hour, &temp, &humidity, &pressure, &wind,
+          &feels_like_temp, &loc_lat, &loc_long, &country));
+
+        printf(
```

Comment:
Consider a compact restructuring, eg:

```        printf(
          "{\n"
          "    ID: %" PRIu64" \n"
          "    day: %s\n"
          "    hour: %" PRIu16" \n"
          "    temp: %" PRIu8" \n"
          "    humidity: %" PRIu8" \n"
          "    pressure: %" PRIu16" \n"
          "    wind: %" PRIu8" \n"
          "    feels like: %" PRIu8" \n"
          "    lat: %" PRIu16" \n"
          "    long: %" PRIu16" \n"
          "    country: %s\n"
          "}\n",
          recno, day, hour, temp, humidity, pressure, wind, feels_like_temp, loc_lat, loc_long,
          country);
    }```

Reply:
I did it that way earlier, however, when I ran s_all it changed it to the current format.